src/components/LoanCards/KivaClassicBasicLoanCard.vue

```diff
@@ -1,7 +1,8 @@
 <template>
 	<div
 		class="tw-flex tw-flex-col"
-		style="min-width: 230px; max-width: 374px; height: 100%;"
+		style="min-width: 230px; height: 100%;"
```

Comment:
Can this line be removed now that it's replicated below?

Reply:
Looks like this duplicate CSS can still be removed.